### PR TITLE
Moving jmx_exporter to a javaagent config and moved kafka-metrics to …

### PR DESCRIPTION
**Pull request by smedefind**
…proper port

### CODE REVIEW

File: recipes/jmx_exporter.rb [hunk 1]
```diff
@@ -1,37 +1,34 @@
...
 
-service 'jmx_exporter' do
-  action [:enable, :start]
-end
-
 consul_definition 'kafka-metrics' do
   type 'service'
   parameters(
-    port: 9200,
+    port: 7071,
```
Comment (zadunn) [inline recipes/jmx_exporter.rb:31]: port 7071??

Comment (jgillis01) [reply recipes/jmx_exporter.rb:31]: I was keeping this on 9200 for the exporter service in the docker image

Comment (smedefind) [reply recipes/jmx_exporter.rb:31]: It's what the javaagent used in the documentation

Comment (smedefind) [reply recipes/jmx_exporter.rb:31]: 9200 also conflicts with ES. So may not want to use it. But also not a problem most likely.

**Pull request merged**